libs/core/src/divider/divider.element.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from 'lit';
+
+export default css`
+  hr {
+    margin-top: 4px;
+    margin-bottom: 4px;
+    border: 0;
```

Comment:
I'd prefer none here to be a bit more explicit about the border being removed (and re-applied for top below)
```suggestion
    border: none;
```